packages/docs/docs/theme/default-blog-theme.md

```diff
@@ -0,0 +1,26 @@
+# Default blog theme
```

Comment:
What about using `Blog Theme` instead? `Default` keyword seems not be correct as it's not the default theme in VuePress, WDYT?

Reply:
Sure.